### PR TITLE
Clarify and simplify ARM term

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "Azure Resource Manager Tools Configuration Settings",
+      "title": "ARM Tools Configuration Settings",
```
Comment (alex-frankel) [inline package.json]: Where does this string get displayed?

Comment (StephenWeatherford) [reply package.json]: In the settings dialog:

image

**Comment by StephenWeatherford** [comment]
Ping.  I still need approval from a dev on aztools.

File: README.dev.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Azure Resource Manager Tools
-A JSON language extension to support Azure Resource Manager deployment templates and template language expressions.
+# Azure Resource Manager ("ARM") Tools
```
Comment (EricJizbaMSFT) [inline README.dev.md]: Yeah I don't think you want me to review this 😂 I can't find any other documentation that does the `("ARM")` thing and it looks a little ugly to me...

None of these have it:
https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview
https://azure.microsoft.com/en-us/features/resource-manager/
https://docs.microsoft.com/en-us/azure/azure-resource-manager/

File: src/AzureRMTools.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ import { UnrecognizedFunctionIssue } from "./UnrecognizedFunctionIssue";
 export async function activateInternal(context: vscode.ExtensionContext, perfStats: { loadStartTime: number; loadEndTime: number }): Promise<void> {
     ext.context = context;
     ext.reporter = createTelemetryReporter(context);
-    ext.outputChannel = vscode.window.createOutputChannel("Azure Resource Manager Tools");
+    ext.outputChannel = vscode.window.createOutputChannel("ARM Tools");
```
Comment (PrashanthCorp) [inline src/AzureRMTools.ts]: I feel @EricJizbaMSFT has a valid point: `ARM` doesn't show up any azure related pages on Bing or Google search, but Azure ARM does. `Azure Resource Manager Tools` seems more explanatory. If it's too long a name, `Azure Tools - ARM` might be less ambiguous.

Q: Is `ARM` a branding Azure uses externally? I've seen MSFT employees use it, and as abbreviations inside Azure Resource Manager pages. Maybe @bmoore-msft can answer this.

Comment (StephenWeatherford) [reply src/AzureRMTools.ts]: This came from legal, see forwarded e-mail.

File: package.json (hunk 1 above)
Comment (EricJizbaMSFT) [reply package.json]: Okay lol but `Tools Configuration Settings` is clearly the problem here - not `Azure Resource Manager`

File: src/AzureRMTools.ts (hunk 3 above)
Comment (EricJizbaMSFT) [reply src/AzureRMTools.ts]: The email still says "ARM" is discouraged. Why can't we just use `Azure Resource Manager` everywhere and follow recommended best practices? It's not _THAT_ long. Also we don't have `Tools` in our other extension names, which will help shorten it

File: package.json (hunk 1 above)
Comment (StephenWeatherford) [reply package.json]: True on that one, looks like it actually never gets displayed.

File: src/AzureRMTools.ts (hunk 3 above)
Comment (StephenWeatherford) [reply src/AzureRMTools.ts]: It gets rather unwieldy in some places, like the language name "Azure Resource Manager Deployment Template", "Azure Resource Manager Language Server", and as the source for diagnostic issues.

Comment (StephenWeatherford) [reply src/AzureRMTools.ts]: The point about "tools" is good.  I guess we could just call it "Azure Resource Manager".  Maybe "Azure Resource Manager Templates"??

Comment (bmoore-msft) [reply src/AzureRMTools.ts]: "ARM" is in formal - in tooling/docs we should be using "Azure Resource Manager".  In GitHub PR comments we can use ARM all we want ;)

**Comment by StephenWeatherford** [comment]
Superseded by #290

**Pull request closed without merging**